File: get-feature-282.py
import sqlite3
import json

conn = sqlite3.connect('features.db')
cursor = conn.cursor()
cursor.execute("SELECT * FROM features WHERE id = 282")
row = cursor.fetchone()

if row:
    columns = [desc[0] for desc in cursor.description]
    result = dict(zip(columns, row))
    print(json.dumps(result, indent=2))
else:
    print('Feature #282 not found')

conn.close()

File: get-feature-282.js
const sqlite3 = require('sqlite3').verbose();
const db = new sqlite3.Database('./features.db');

db.get("SELECT * FROM features WHERE id = 282", (err, row) => {
  if (err) {
    console.error('Error:', err);
    process.exit(1);
  }
  if (row) {
    console.log(JSON.stringify(row, null, 2));
  } else {
    console.log('Feature #282 not found');
  }
  db.close();
});
